src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ODataEntityReferenceLinkWrapper.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.OData;
+
+namespace Microsoft.AspNet.OData.Formatter.Deserialization
+{
+    /// <summary>
+    /// Encapsulates an <see cref="ODataEntityReferenceLink"/> .
+    /// </summary>
+    public sealed class ODataEntityReferenceLinkWrapper : ODataItemBase
```

Comment:
@ElizabethOkerio This class looks similar to `ODataEntityReferenceLinkBase`. Do we really need to introduce a new class?

Reply:
we wanted some form of consistency in the naming..Most of the wrappers have the wrapper word on them.. Can do away with the other class.

Reply:
That would be a breaking change since the class is `public`...

Reply:
But you could leave the old class until a major release opportunity presents itself